src/stream_base-inl.h

```diff
@@ -92,7 +92,7 @@ WriteWrap* WriteWrap::New(Environment* env,
   size_t storage_size = ROUND_UP(sizeof(WriteWrap), kAlignSize) + extra;
   char* storage = new char[storage_size];
 
-  return new(storage) WriteWrap(env, obj, wrap, cb);
+  return new(storage) WriteWrap(env, obj, wrap, cb, extra);
```

Comment:
Over thinking the situation, about how it may be possible in the future that having the size of `extra` would be useful... Dumb in retrospect. Will fix.